### ISSUE TITLE
[Routing View] Routing events showing up inconsistently

### ISSUE THREAD

**Issue by hieblmi** [edouardparis/lntop#71]
Under the routing menu F2 -> ROUTING the routing events don't seem to persist for long or show up inconsistently. Is this behavior expected or do others see this as a bug too?

Ideally a list of routing events should be appended to with the size of the buffer/file configurable in the `config.toml`

**Comment by edouardparis** [comment]
Yes, the features was introduced without caching. I guess people run lntop in a screen or tmux window. 
The cache file configurable in the config.toml is a good idea. Do you think that a json object for each routing event per line is a good format ? or should we use csv ?

**Comment by hieblmi** [comment]
That's how I run it :-). I think json would be fine so one could run queries with jq if needed. Why were you thinking about csv?

**Comment by hieblmi** [comment]
@edouardparis I've submitted a PR https://github.com/edouardparis/lntop/pull/101 for a forwarding history tab so we could discuss whether this ticket is still needed unless we want to introduce caching for failed forwards in the routing tab. Curious about your thoughts.
